### PR TITLE
Fix nightlies page

### PR DESCRIPTION
**Pull request by straight-shoota**
The nightlies page was broken since #570 because it didn't make the transform.
This PR extracts package data for nightlies and builds the nighlies page in the same was as the install page.
Als improves readability on the normal package data file.

### CODE REVIEW

File: _pages/install/nightlies.md [hunk 1]
```diff
@@ -3,134 +3,31 @@ page_title: Nightly Builds
...
-        <a href="https://github.com/neatorobito/scoop-crystal" title="Scoop repository for Crystal on GitHub" class="info">{% include icons/info.svg %}</a>
-      </div>
-    </div>
-  </div>
-</div>
+{% include pages/install/section.html os="Windows" channel="nightly" %}
 
 ## Docker
 
 Nightly builds are available on the `nightly` tag on the [Docker repository of Crystal](https://hub.docker.com/r/crystallang/crystal/).
```
Comment (beta-ziliani) [inline _pages/install/nightlies.md:24]: This is now kind of redundant, isn't it?

**Review by beta-ziliani — APPROVED** [review]
Shouldn't "nightlies" and "from source" be in the TOC?

File: _pages/install/nightlies.md (hunk 1 above)
Comment (straight-shoota) [reply _pages/install/nightlies.md:24]: Yes, it'll be removed in a follow-up.

**Comment by straight-shoota** [comment]
> Shouldn't "nightlies" and "from source" be in the TOC?

Yes, there's a follow-up coming for that.

**Pull request merged**